src/SkirtBrim.cpp

```diff
@@ -84,7 +84,7 @@ void SkirtBrim::getFirstLayerOutline(SliceDataStorage& storage, const size_t pri
     }
 }
 
-coord_t SkirtBrim::generatePrimarySkirtBrimLines(const coord_t start_distance, size_t primary_line_count, const coord_t primary_extruder_minimal_length, const Polygons& first_layer_outline, Polygons& skirt_brim_primary_extruder)
+coord_t SkirtBrim::generatePrimarySkirtBrimLines(const coord_t start_distance, size_t& primary_line_count, const coord_t primary_extruder_minimal_length, const Polygons& first_layer_outline, Polygons& skirt_brim_primary_extruder)
```

Comment:
Assuming the Added & indicates that primarly_line_count lives in the heap. Is this a bug fix? Otherwise I would expect additional changes that refactor this variable to the heap

Reply:
Passing by reference (or pointer, for that matter) doesn't always indicate that something lives on the heap. You can have references (and pointers) to a memory location to the stack as well.

(Though I'm not sure heap and stack are used very much as terms any more, they've become inaccurate/legacy.)